src/extraction/index.js

```diff
@@ -57,8 +74,8 @@ async function sendProductToBackground(extractedProduct) {
  * Checks to see if any product information for the page was found,
  * and if so, sends it to the background script.
  */
-async function attemptExtraction() {
-  const extractedProduct = extractProduct();
+async function attemptExtraction(isBackgroundUpdate) {
+  const extractedProduct = extractProduct(isBackgroundUpdate);
```

Comment:
Instead of passing this parameter through a bunch of methods, if we move the `isBackgroundUpdate` code into a function, we can call it again when we need to log the value a second time. We could also assign it to a global variable, as whether the page is a background update or not is not expected to change within this script.